app/src/main/kotlin/com/santaclose/app/restaurant/resolver/dto/RestaurantAppQueryResolver.kt

```diff
@@ -0,0 +1,35 @@
+package com.santaclose.app.restaurant.resolver.dto
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.server.operations.Query
+import com.santaclose.app.auth.directive.Auth
+import com.santaclose.app.restaurant.service.RestaurantAppQueryService
+import com.santaclose.lib.entity.appUser.type.AppUserRole
+import com.santaclose.lib.logger.logger
+import com.santaclose.lib.web.error.toGraphQLException
+import javax.validation.Valid
+import org.springframework.stereotype.Component
+import org.springframework.validation.annotation.Validated
+
+@Component
+@Validated
+class RestaurantAppQueryResolver(private val restaurantAppQueryService: RestaurantAppQueryService) :
```

Comment:
```suggestion
class RestaurantAppQueryResolver(private val restaurantAppQueryService: RestaurantAppQueryService,) :
```

kotlin 쪽 Linter 가 좋은 편은 아니라서 생성자 끝에 `,` 를 하나 넣어주면 좋을거 같습니다.
그러면 저장 시 프로퍼티 선언부가 한줄 아래로 내려가서 나중에 새로운 프로퍼티 추가할 때 편해진다고 생각합니다.